lxd/storage_volumes_snapshot.go

```diff
@@ -731,16 +728,12 @@ func autoCreateCustomVolumeSnapshotsTask(d *Daemon) (task.Func, task.Schedule) {
 
 			// If there is more than one node (clustering), a stable random node is chosen to perform the snapshot.
 			if len(nodes) > 1 {
-				seed := fmt.Sprintf("%d", v.ID)
-
-				hash := fnv.New64a()
-				_, err = io.WriteString(hash, seed)
+				nodeIDIndex, err := util.GetStableRandomInt64FromList(int(v.ID), availableNodeIDs)
```

Comment:
@tomponline this is incorrect. `GetStableRandomInt64FromList` returns an actual value from the list, not just an index. It therefore should read
```go
selectedNodeID, err := util.GetStableRandomInt64FromList(int(v.ID), availableNodeIDs)
```

Reply:
@monstermunchkin got you thanks. fixed